src/storage/src/hummock/sstable_store.rs

```diff
@@ -372,6 +249,65 @@ impl SstableStore {
     pub fn clear_block_cache(&self) {
         self.block_cache.clear();
     }
+
+    pub async fn load_table(
+        &self,
+        sst_id: HummockSSTableId,
+        stats: &mut StoreLocalStatistic,
+        load_data: bool,
+    ) -> HummockResult<TableHolder> {
+        stats.cache_meta_block_total += 1;
+        loop {
+            let entry = self
+                .meta_cache
+                .lookup_with_request_dedup::<_, HummockError, _>(sst_id, sst_id, || async {
+                    stats.cache_meta_block_miss += 1;
+                    let path = self.get_sst_meta_path(sst_id);
+                    let buf = self
+                        .store
+                        .read(&path, None)
+                        .await
+                        .map_err(HummockError::object_io_error)?;
+                    let mut size = buf.len();
+                    let meta = SstableMeta::decode(&mut &buf[..])?;
+                    let blocks = if load_data {
+                        size = meta.estimated_size as usize;
+                        self.load_data(sst_id, &meta.block_metas).await?
+                    } else {
+                        vec![]
+                    };
+                    let sst = Box::new(Sstable {
+                        id: sst_id,
+                        meta,
+                        blocks,
+                    });
```

Comment:
We should add a method (`new_with_blocks`) to construct Sstable with non-empty blocks in  `Sstable` and avoid using struct initialization.

Reply:
ok